src/plugins/wizard/public/application/contributions/containers/data_tab/items/use/use_dropbox.tsx

```diff
@@ -154,17 +164,36 @@ export const useDropbox = (props: UseDropboxProps): DropboxProps => {
     onDropField
   );
 
-  const isValidDropField = useMemo(() => {
-    if (!dragData) return false;
+  useEffect(() => {
```

Comment:
This hook should probably be a separate file, partly because it looks like it will require substantial unit tests to verify all the edge cases.

Reply:
Good call. Ill leave it in here for this PR. I am yet to add another validation for min and max values. Once i do that, ill move it out with some tests.